packages/nuxt/src/runtime/plugins/sentry.server.ts

```diff
@@ -1,4 +1,4 @@
-import { captureException } from '@sentry/node';
+import * as Sentry from '@sentry/node';
```

Comment:
super-l/hint: It's totally fine to make a namespace (`import * as`) or named imports. No need to change it back, just fyi :)